spec/index.bs

```diff
@@ -1500,9 +1509,25 @@ dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
     float lodMinClamp = 0;
     float lodMaxClamp = 0xffffffff; // TODO: What should this be? Was Number.MAX_VALUE.
     GPUCompareFunction compare = "never";
+    boolean compareEnable = false;
```

Comment:
Why not instead use `undefined` for `compareEnable` as a tag value?

Reply:
Let's not make `false` and `undefined` have different semantics, but a 3-value enum could maybe work.

Reply:
Wait, nvm, sorry, I think you meant `undefined` for `compare`. That would work.